src/Payment.php

```diff
@@ -19,22 +19,22 @@
 
 class Payment implements PaymentInterface
 {
-    const FEE_FIXED = 'fixed';
-    
-    const FEE_PERCENTAGE = 'percentage';
+    private const FEE_FIXED = 'fixed';
+
+    private const FEE_PERCENTAGE = 'percentage';
```

Comment:
Those consts should remain public as you can use them when instantiating Payment object. Although maybe enum would fit better here.

Reply:
Ah, didnt notice you use them elsewhere. Still, it should be said clearly, they are public a not by default ;) Anyway, will not change that. if you want you can copy what you want from this PR